packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -25,41 +26,135 @@ const storybookMeta: Meta<ButtonProps> = {
   argTypes: {
     isDisabled: {
       control: "boolean",
+      description: "If `true`, the button is disabled",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: "",
+      },
     },
     isFullWidth: {
       control: "boolean",
+      description:
+        "If `true`, the button will take up the full width available",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: "",
+      },
     },
```

Comment:
Something about this seems wrong. Do we need to set `defaultValue: ""`? I thought that's supposed to be optional.